notebooks/SITCOMTN_101_FCU_investigation.ipynb

```diff
@@ -0,0 +1,1216 @@
+{
```

Comment:
<p>I see a few duplicated imports in this cell. Could you review it, please? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-101/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
fixed

Reply:
done